src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -529,8 +533,8 @@ def get_historical_consent_report(
             ),
             PrivacyPreferenceHistory.url_recorded.label("url_recorded"),
             PrivacyPreferenceHistory.user_agent.label("user_agent"),
-            PrivacyPreferenceHistory.privacy_experience_history_id.label(
-                "privacy_experience_history_id"
+            PrivacyPreferenceHistory.privacy_experience_id.label(
+                "privacy_experience_id"
```

Comment:
Similarly the historical report now surfaces the experience id instead of the experience history id.  This isn't that informative of a property, as it only stores the region and component, but I want to record it in case we need to filter on it.  This was, after all, the primary id of the resource that housed everything we displayed to the end user.